compiler/damlc/tests/daml-test-files/InterfaceMissingMethod.daml

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR range=12:10-14:14;Template T is missing method m for interface InterfaceMissingMethod:I
```

Comment:
extra points if you get this to point to the `implements` clause but I’m happy with just the template for now.